vm/vmimpl/vmimpl.go

```diff
@@ -107,6 +109,21 @@ func (err BootError) BootError() (string, []byte) {
 	return err.Title, err.Output
 }
 
+// By default, all Pool.Create() errors are related to infrastructure problems.
+// VerboseInfraError is to be used when we want to attach more output.
+type VerboseInfraError struct {
```

Comment:
Since there is no NonVerboseInfraError, I would call this just InfraError. Also consistent with BootError.